### DISCUSSION TITLE
Make handling of tab key smarter

### DISCUSSION THREAD

**Discussion by speich** [digitalfortress-tech/typeahead-standalone#25] [Ideas]
Using preventDefault() for the tab key in https://github.com/digitalfortress-tech/typeahead-standalone/blob/master/src/typeahead-standalone.ts#L435 breaks form usability (I normally use tab to go from one form field to the next). 

How about only calling preventDefault() when there is actually a value to set (e.g. when the list is open and has matching values). This way, the first pressing of tab key completes, the second time one presses the tab key, the default is executed and the browser moves the cursor to the next field.

**Comment by niketpathak** [comment]
yes, good point. So the default action of selecting a suggestion will take place only while the list of suggestions is open. This will be implemented in the next release.... Thank you!

**Reply by niketpathak** [reply]
Hello, this has been fixed in version [4.7.0](https://github.com/digitalfortress-tech/typeahead-standalone/releases/tag/v4.7.0)
